integration/tests/sign-out-smoke.test.ts

```diff
@@ -45,4 +45,68 @@ testAgainstRunningApps({ withEnv: [appConfigs.envs.withEmailCodes] })('sign out
 
     await mainTab.po.expect.toBeSignedOut();
   });
+
+  test('sign out destroying client', async ({ page, context }) => {
+    const u = createTestUtils({ app, page, context });
+    await u.po.signIn.goTo();
+    await u.po.signIn.setIdentifier(fakeUser.email);
+    await u.po.signIn.continue();
+    await u.po.signIn.setPassword(fakeUser.password);
+    await u.po.signIn.continue();
+    await u.po.expect.toBeSignedIn();
+    await u.page.goToAppHome();
+
+    await u.page.waitForSelector('p[data-clerk-id]', { state: 'attached' });
+
+    await u.page.evaluate(async () => {
+      await window.Clerk.signOut();
+    });
+
+    await u.po.expect.toBeSignedOut();
+    await u.page.waitForSelector('p[data-clerk-id]', { state: 'detached' });
+    await u.page.waitForSelector('p[data-clerk-session]', { state: 'detached' });
+  });
 });
+
+testAgainstRunningApps({ withEnv: [appConfigs.envs.withEmailCodes_persist_client] })(
+  'sign out smoke test @generic',
```

Comment:
Would be the same name otherwise (as the first one in this file)

```suggestion
  'sign out with persistClient smoke test @generic',
```